src/devices/tuya.ts

```diff
@@ -11447,6 +11447,57 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_fhvdgeuh']),
+        model: 'TS0601',
+        vendor: 'Tuya',
+        description: 'Din rail switch with power monitoring and threshold settings',
+        fromZigbee: [tuya.fz.datapoints, fz.metering],
```

Comment:
Is `fz.metering` really needed?

Reply:
Isn't fz.metering needed for PhaseA to current, voltage and power? Or is that tuya.valueConverter.phaseVariant2 enough?

Reply:
Guess you are right, looking at the code in fromZigbee.ts line 730. I'll remove it